waku/v2/protocol/relay/waku_relay.go

```diff
@@ -227,9 +227,38 @@ func (w *WakuRelay) Subscribe(ctx context.Context, topic *Topic) (*Subscription,
 	return subscription, nil
 }
 
+func (w *WakuRelay) Unsubscribe(ctx context.Context, topic *Topic) error {
+	if _, ok := w.topics[*topic]; !ok {
+		return fmt.Errorf("topics %s is not subscribed", (string)(*topic))
+	}
+	log.Info("Unsubscribing from topic ", *topic)
+	delete(w.topics, *topic)
+
+	for _, sub := range w.subscriptions[*topic] {
```

Comment:
Should we also delete the subscriptions? (delete(w.subscriptions, topic)
I'm not sure what would happen if we do that and there are active subscriptions being used

Reply:
I didn't delete them because of this:
```
case <-subscription.quit:
			if w.bcaster != nil {
				w.bcaster.Unregister(subscription.C) // Remove from broadcast list
			}
			// TODO: if there are no more relay subscriptions, close the pubsub subscription
```
Shall we delete them there?